elyra/metadata/metadata_app.py

```diff
@@ -908,7 +1034,7 @@ def __init__(self, **kwargs):
         args = kwargs.get("argv", [])
         if len(args) > 0:
             # identify commands that can operate on deprecated schemaspaces
-            include_deprecated = args[0] not in ["install", "create", "update"]
+            include_deprecated = args[0] not in ["install", "create", "update", "import"]
```

Comment:
I don't think we should allow imports into deprecated schemaspaces.

Reply:
Hey @kevin-bates the code ensures that the commands in the list are NOT allowed to work with deprecated schemas right? So - import should be included in the list. I also have a test (test_import_deprecated_schemaspace) that verifies that import fails for "component-registries"

Reply:
Hmmm - I can't get even a command like list to work with "component-registries". I thought that was a valid "deprecated" schemaspace.

Reply:
Hi @sandhya-nayak,

> the code ensures that the commands in the list are NOT allowed to work with deprecated schemas right

I'm sorry, I missed the `not` on the `in` clause and didn't stop to think that we need to **prevent "writes"** into deprecated schemaspaces - which are exactly the kinds of operations listed.  I apologize.

> can't get even a command like list to work with "component-registries". I thought that was a valid "deprecated" schemaspace

Component-registries were removed (_completely deprecated_) in #2497, so what you're seeing sounds like expected behavior.

Reply:
Ok @kevin-bates - so it looks like the "test_import_deprecated_schemaspace" isn't really needed, as it basically treats component-registries as a bad schemaspace (which is being covered by "test_import_bad_schemaspace"). Shall I remove that test_import_deprecated_schemaspace?

Reply:
That makes sense - thank you.  Schemaspce deprecation will be a rare (and temporary) event - as will be any associated tests.